deployment/helm/nri-plugins/topology_aware_values.yaml

```diff
@@ -0,0 +1,43 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
```

Comment:
By default the image that Helm chart will be setting on the final deployment is
`ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware:unstable`